src/chrome/content/rules/WikiFur.xml

```diff
@@ -1,8 +1,12 @@
 <ruleset name="WikiFur">
   <target host="wikifur.com" />
   <target host="*.wikifur.com" />
```

Comment:
Please avoid using wildcards and enumerate subdomains using domain enumeration tools, such as [Amass](https://github.com/OWASP/Amass).

```suggestion
	<target host="www.wikifur.com" />
	<target host="be.wikifur.com" />
	<target host="be-tarask.wikifur.com" />
	<target host="board.wikifur.com" />
	<target host="by.wikifur.com" />
	<target host="cs.wikifur.com" />
	<target host="da.wikifur.com" />
	<target host="de.wikifur.com" />
	<target host="dk.wikifur.com" />
	<target host="dumps.wikifur.com" />
	<target host="en.wikifur.com" />
	<target host="eo.wikifur.com" />
	<target host="es.wikifur.com" />
	<target host="et.wikifur.com" />
	<target host="fr.wikifur.com" />
	<target host="fursuit.wikifur.com" />
	<target host="hu.wikifur.com" />
	<target host="it.wikifur.com" />
	<target host="ja.wikifur.com" />
	<target host="ko.wikifur.com" />
	<target host="nb.wikifur.com" />
	<target host="nl.wikifur.com" />
	<target host="no.wikifur.com" />
	<target host="ny.wikifur.com" />
	<target host="pl.wikifur.com" />
	<target host="pool.wikifur.com" />
	<target host="pt.wikifur.com" />
	<target host="ru.wikifur.com" />
	<target host="stats.wikifur.com" />
	<target host="tl.wikifur.com" />
	<target host="uk.wikifur.com" />
	<target host="zh.wikifur.com" />
```